src/fhirConfig.ts

```diff
@@ -23,8 +23,21 @@ export interface OAuthStrategy {
     revokeUrl?: string;
 }
 
+/**
+ * http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#using-well-known
+ */
+export interface SmartStrategy extends OAuthStrategy {
+    capabilities: string[];
+    scopesSupported?: string[];
```

Comment:
Could make a deterministic type but again it may be too rigid?

Reply:
In terms of one-way-door vs two-way-door decisions, I'm thinking keeping it a string[] is a two way door decision while making it into a specific type is a one way door decision.

Leaving it as a string[] now will still allow us to narrow the scope in the future if we get customer feedback for that in the future. That will only be a minor version bump. However, scoping it down now will require a major version bump in the future as that is breaking change.